newdle/export.py

```diff
@@ -27,10 +28,14 @@ def _generate_answers_for_export(newdle):
 
 def export_answers_to_csv(newdle):
     rows = _generate_answers_for_export(newdle)
-    csv = '\n'.join([','.join(row) for row in rows])
     buffer = BytesIO()
-    buffer.write(csv.encode('utf-8-sig'))
+    output = StringIO()
+    writer = csv.writer(output, dialect='unix')
+    writer.writerows(rows)
+    buffer.write(output.getvalue().encode('utf-8-sig'))
     buffer.seek(0)
+    output.close()
```

Comment:
Check the `csv_text_io_wrapper` we use in Indico to avoid this mess with two separate buffers.